sse2neon.h

```diff
@@ -7521,18 +7521,25 @@ FORCE_INLINE __m128d _mm_dp_pd(__m128d a, __m128d b, const int imm)
     // Generate mask value from constant immediate bit value
     const int64_t bit0Mask = imm & 0x01 ? UINT64_MAX : 0;
     const int64_t bit1Mask = imm & 0x02 ? UINT64_MAX : 0;
-    const int64_t bit4Mask = imm & 0x10 ? UINT64_MAX : 0;
-    const int64_t bit5Mask = imm & 0x20 ? UINT64_MAX : 0;
     // Conditional multiplication
-    __m128d mul = _mm_mul_pd(a, b);
-    const __m128d mulMask =
-        _mm_castsi128_pd(_mm_set_epi64x(bit5Mask, bit4Mask));
-    __m128d tmp = _mm_and_pd(mul, mulMask);
+    __m128d mul;
+#if defined(__aarch64__)
```

Comment:
So, no more bit mask?

Reply:
The bit mask for conditional multiplication is no longer required.